index.html

```diff
@@ -1811,6 +1811,125 @@ <h3>Interface <dfn>ReceiveStream</dfn></h3>
     </div>
   </section>
 
+  <section id="http3-transport*">
+    <h2><dfn>Http3Transport</dfn> Interface</h2>
+    <p>The <code>Http3Transport</code> is ***
+    </p>
+    <section id="http3-transport-overview*">
+      <h3>Overview</h3>
+      <p>An <code><a>Http3Transport</a></code> is a
+      <code>UnidirectionalStreamsTransport</code>, a
+      <code>BidirectionalStreamsTransport</code>, and a
+      <code>DatagramTransport</code>.
+      SendStreams, ReceiveStreams, and BidirectionalStreams are implemented with
+      HTTP/3 streams as defined in [[WEB-TRANSPORT-HTTP3]]. Datagrams are
+      implemented with QUIC datagrams as defined in [[WEB-TRANSPORT-HTTP3]].
+      </p>
+    </section>
+    <section id="http3-transport-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+[ Constructor (DOMString url), Exposed=Window]
+interface Http3Transport {
+};
+
+Http3Transport includes UnidirectionalStreamsTransport;
+Http3Transport includes BidirectionalStreamsTransport;
+Http3Transport includes DatagramTransport;
+Http3Transport includes WebTransport;</pre>
+      </div>
+        <section>
+          <h2>Constructors</h2>
+          When the <code><a>Http3Transport</a></code> constructor is invoked,
+          the user agent <em class="rfc2119" title="MUST">MUST</em> run the
+          following steps:
+          <ol>
+            <li>
+              Let <var>transport</var> be a newly constructed
+              <code><a>Http3Transport</a></code> object with state "connecting".
+            </li>
+            <li>Let <var>transport</var> have a <dfn>[[\OutgoingStreams]]</dfn>
+              internal slot representing a sequence of <code><a>OutgoingStream</a></code>
+              objects, initialized to empty.
+            </li>
+            <li>Let <var>transport</var> have a <dfn>[[\IncomingStreams]]</dfn>
+              internal slot representing a sequence of <code><a>IncomingStream</a></code>
+              objects, initialized to empty.
+            </li>
+            <li>
+                Let <var>transport</var> have a <dfn>[[\WebTransportState]]</dfn>
+                internal slot, initialized to <code>"connecting"</code>.
+            </li>
+            <li>Let <var>transport</var> have a <dfn>[[\ReceivedDatagrams]]</dfn>
+                internal slot representing a queue of <code>Uint8Array</code>, initialized to empty.
+            </li>
+            <li>Let <var>transport</var> have a <dfn>[[\ReceiveDatagramsPromise]]</dfn>
+                internal slot representing a <code>Promise&lt;sequence&lt;Uint8Array&gt;&gt;?</code>,
+                initialized to null.
+            <li>Run these steps in parallel:
+              <ol>
+                <li>
+                  Either establish an HTTP/3 connection or reuse an existing HTTP/3
```

Comment:
Do we need to specify how we establish an HTTP/3 connection or reuse one here?

Reply:
I'll punt to the IETF doc :)